### PR TITLE
Show all files on repo home, implements #1508

### PR DESCRIPTION
**Pull request by montogeek**
Implements https://github.com/sindresorhus/refined-github/issues/1508.

Not 100% sure about the file name, other options:
- click-show-all-files
- click-all-files

Tested it locally, it worked.

### CODE REVIEW

**Comment by bfred-it** [comment]
`always-show-all-files`

**Comment by montogeek** [comment]
@bfred-it Updated

File: source/features/always-show-all-files.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+
+export default async function () {
+	const button = select('.js-details-target');
```
Comment (bfred-it) [inline source/features/always-show-all-files.js]: This selector is too generic. However I'm not seeing the button anymore, did GitHub remove it already?

Comment (montogeek) [reply source/features/always-show-all-files.js]: Yep, it is temporal.
https://twitter.com/clarkbw/status/1037729090101334017

**Comment by bfred-it** [comment]
This can be closed, GitHub dropped the feature

cc @busches

**Pull request closed without merging**